sphinx/screens/profile/profile/src/main/java/chat/sphinx/profile/ui/ProfileFragment.kt

```diff
@@ -200,6 +202,35 @@ internal class ProfileFragment: SideEffectFragment<
             }
 
             includeProfileAdvancedContainerHolder.apply {
+                editTextProfileAdvancedContainerServerUrl.setOnFocusChangeListener { _, hasFocus ->
+                    if (hasFocus) {
+                        return@setOnFocusChangeListener
+                    }
+                    onStopSupervisor.scope.launch(viewModel.mainImmediate) {
+                        viewModel.updateRelayUrl(
+                            editTextProfileAdvancedContainerServerUrl.text.toString()
```

Comment:
utilization of the `Any?.toString()` extension function (as is with an `EditTextView.text`) here will produce a string vallue of "null". Nullability check needs to occur after the `.text` value (`EditText.text?.toString()` to produce a null value, or an empty string)